.gitignore

```diff
@@ -1,3 +1,6 @@
+# Glade backup file
+*.glade~
```

Comment:
This should not be added to git.

Reply:
Why not? `.gitignore` is used to ignore build files and temporary files.